src/utils/entries/entries.ts

```diff
@@ -0,0 +1,5 @@
+import { Entries } from './entries.types';
+
+export function entries<TEntries>(obj: TEntries): Entries<TEntries> {
```

Comment:
Co robi ta funkcja? Bo nie rozumiem po opisie?

Reply:
To jest customowa funkcja, która robi dokładnie to samo co Javascriptowe `Object.entries()`, tylko daje `entries` w odpowiednim typie. Jest mi ona potrzebna w kilku miejscach, żeby uniknąć oszukiwania TypeScripta. Oryginalny `Object.entreis` działa niedokładnie w Typescripcie, bo np. w takim przypadku: `Object.entries({ x:1, y:2 })` zwraca entries jako `[key: string, value: number]`, a ja potrzebowałem, żeby mi dawał `[key: 'x' | 'y', value: number]`